deployer/scripts/util.sh

```diff
@@ -546,3 +546,23 @@ function wait_for_fluentd_to_catch_up() {
     echo END wait_for_fluentd_to_catch_up at `date -u --rfc-3339=ns`
     return $rc
 }
+
+docker_uses_journal() {
+    # note the unintuitive logic - in this case, a 0 return means true, and a 1
+    # return means false
+    # need to be able to handle cases like
+    # OPTIONS='--log-driver=json-file ....' # or use --log-driver=journald
+    # if "log-driver" is set in /etc/docker/daemon.json, assume that it is
+    # authoritative
```

Comment:
In this case, the `elif grep -q '^[^#].*"log-driver":' /etc/docker/daemon.json` at line 560 will return `true`, so the flow will go to line 561, where the `if grep -q '^[^#].*"log-driver":.*journald' /etc/docker/daemon.json` will return `false`.  The flow will then fall through to line 567 and return `false`.  Which means, if `log-driver` is set in `/etc/docker/daemon.json`, but the value is not `journald`, return `false`, which is the expected behavior.